Content.Server/Atmos/EntitySystems/GasTankSystem.cs

```diff
@@ -396,13 +401,17 @@ public void CheckStatus(Entity<GasTankComponent> ent)
         /// </summary>
         private void OnAnalyzed(EntityUid uid, GasTankComponent component, GasAnalyzerScanEvent args)
         {
-            args.GasMixtures ??= new List<(string, GasMixture?)>();
-            args.GasMixtures.Add((Name(uid), component.Air));
+            if (TryComp<InternalAirComponent>(uid, out var internalAir))
+            {
+                args.GasMixtures ??= new List<(string, GasMixture?)>();
+                args.GasMixtures.Add((Name(uid), internalAir.Air));
+            }
         }
 
         private void OnGasTankPrice(EntityUid uid, GasTankComponent component, ref PriceCalculationEvent args)
         {
-            args.Price += _atmosphereSystem.GetPrice(component.Air);
+            if (TryComp<InternalAirComponent>(uid, out var internalAir))
+                args.Price += _atmosphereSystem.GetPrice(internalAir.Air);
         }
```

Comment:
This should be on internalair not here as it doesn't even need gastank.

Reply:
done.